Report.md

```diff
@@ -31,6 +31,14 @@ Problem Definition
 ## **Assumptions**
  5. List assumptions you have made in the systems analysis.
 
```

Comment:
Maybe write an introductory line like "Over the course of system analysis, there were a few assumptions we had to make in order to understand the system as a whole. These assumptions are listed below:"
[I was actually doing this at the same time so I just stole it ahaha]

Reply:
Sure, I will add that in.  Thanks!